### DISCUSSION TITLE
Questions regarding this project

### DISCUSSION THREAD

**Discussion by Joe23232** [hecrj/iced#859] [Q&A] [answered]
Hi, I have some questions regarding this project.

1. Is it supported on Windows, Linux x11 and Linux wayland?
2. Is it also supported on Android if I used something like [`cargo-mobile`](https://github.com/BrainiumLLC/cargo-mobile) or [`android-ndk-rs`](https://github.com/rust-windowing/android-ndk-rs)
3. Is it retained or immediate graphics?
4. So if I need to update a specific part of the window would it or would it refresh the entire window?
5. Using WASM Rust, can I make web applications using `iced`?
6. How does this crate differ to [`orbtk`](https://github.com/redox-os/orbtk)?

**Comment by 13r0ck** [comment, marked as answer]
1. Yes
2. No. [It is in the works though!](https://github.com/hecrj/iced/pull/830). But please be patient, I don't expect it any time soon.
3. Not 100% sure, but I believe retained
4. N, Iced uses a primitive tree
5. [Yes](https://github.com/hecrj/iced/tree/master/web), It is still very much a WIP though.
6. Iced is inspired by the Elm architecture, which helps structure easier to read, and faster built/modify applications

**Reply by martin-braun** [reply]
Hi, I have two more additional questions down the line that match the other questions and I would appreciate a follow up.

7. Are there also plans on bringing Iced to iOS?
8. Does the web target produce a proper DOM (instead of drawing on a canvas)?

**Reply by 13r0ck** [reply]
No and no

**Comment by Joe23232** [comment]
Right I see, thanks mate :)
